backstage/templates/scaffolder/add-repo/content/.gitignore

```diff
@@ -0,0 +1,35 @@
+# Local .terraform directories
```

Comment:
this gitignore doesn't make sense on a generic repo.

Reply:
think I should just remove it? I didn't see anything on this list that looked like a "git fit" 😆

Reply:
if there are any common IDE-specific things, go for it. Otherwise, we could just have a blank .gitignore as a placeholder